assets/js/swedbank-pay-design-guide-theme.js

```diff
@@ -20,54 +20,95 @@
 
 // Initialize sidebar navigation scroll activation
 (function () {
+    const pathHash = window.location.pathname + window.location.hash;
+    const originPath = window.location.origin + window.location.pathname
+    const activeGroup = [...document.querySelectorAll(".nav-group")]
+    .filter(group => group.querySelector(`[href="${pathHash}"], [href^="${originPath}#"]`))[0];
+    const activeSubgroup = [...document.querySelectorAll(".nav-subgroup")].filter(group => group.querySelector(`[href="${pathHash}"]`))
+
+    activeGroup.classList.add("active")
+    activeSubgroup.map(subgroup => subgroup.classList.add("active"))
+
     var headings = document.querySelectorAll("h2");
-    var tocLinks = document.querySelectorAll("nav.doc-toc ul li a");
+    var tocLinks = document.querySelectorAll("nav.sidebar-nav .nav-subgroup.active .nav-leaf");
+
+    if (tocLinks.length == 0) {
+        var tocLinks = document.querySelectorAll("nav.sidebar-nav .nav-group.active .nav-leaf");

```

Comment:
Codacy found an issue: ['tocLinks' is already defined.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5659803)